### ISSUE TITLE
Mismatch between number of training data and labels

### ISSUE THREAD

**Issue by yingzhi116** [yizhuoyang/AV-FDTI#5]
Hi there- Thank you for your impressive work, I'm working to reproduce and promote your work, but I'm meeting a little difficulty.
I have downloaded the ROSBag data of Mavic2-M300 and the corresponding GT.bag file in MMAUD. And I also follow your method to align audio and image timestamps.  But I found the number of GT(both absolute and relative) is much smaller than audio and image data. So I want to know how to build the labels of the dataset before the training. I would really appreciate it if you answered my question.

**Comment by DanielCohenn** [comment]
@yingzhi116 Hey, have you managed to algin the timestamps ? I am going through the same process and I see that the authors have not answered you yet. If you managed to extract and align the dataset I would be happy to know how you did it !

**Comment by snakehaihai** [comment]
There’s no perfect solution, as different sensors operate at different time frequencies, making alignment difficult. The simplest approach is to represent the trajectory using linear interpolation or splines, so you can sample the ground truth at the exact times when each sensor captures data.

**Comment by DanielCohenn** [comment]
@snakehaihai 
Oh okey,
So I have to extract each sensor data from the .bag files and then plot the trajectory of the UAV.
So we would need to interpolate and plot the trajectory given to us from the ground truth and then sample the trajectory to extract data from the audio and vision sensors ?

**Comment by snakehaihai** [comment]
thats the easiest way to get higher accuracy GT
